packages/integration-tests/projects/suite-web/tests/suite/database-migration.test.ts

```diff
@@ -1,46 +1,139 @@
 // @group:migrations
 // @retry=2
 
+import { rerouteMetadataToMockProvider, stubOpen } from '../../stubs/metadata';
+
 describe('Database migration', () => {
     beforeEach(() => {});
 
-    Cypress.env('TEST_URLS').forEach((testUrl: string, index: number) => {
+    const testUrls = ['/release/22.7/web', '/develop/web'];
+    // const testUrls = ['/release/22.7/web'];
+    const provider = 'dropbox';
+    const hiddenWalletSelector = '[data-test^="@switch-device/wallet-on-index"]';
+    const testData = {
+        accPhrase: 'doggo je dobros',
+        walletLabel: 'spending wallet',
+    };
+
+    testUrls.forEach((testUrl: string, index: number) => {
+        // Cypress.env('TEST_URLS').forEach((testUrl: string, index: number) => {
         const keepDB = index > 0;
 
-        it(index === 0 ? `From ${testUrl}` : `To ${testUrl}`, () => {
+        /**
+         * Test case:
+         * 1. in in the older version of the app navigate to settings/device and verify settings state
+         * 2. navigate to the wallet overview and add the hidden QA wallet
+         * 3. navigate to Accounts, click on the first btc account and verify tx
+         * 4. remember the wallet
+         * 5. navigate to the newer version of the app
+         * 6. navigate to the wallet overview and verify that the remembered wallet is still present
+         * 7. navigate to Accounts, click on the first btc account and verify that txs are still present
+         * and the same as in the old app
+         * 8. click on the Recieve btn and verify that the address displays normally
+         * 9. click on the Send btn and verify that the form is not broken (visually?)
+         * 10. verify that a dark theme is still applied
+         */
+        it(index === 0 ? `From ${testUrl}` : `To ${testUrl}`, function () {
+            //
+            // Test data preparation
+            //
+
             cy.viewport(1080, 1440);
             cy.task('startEmu', { wipe: true });
-            cy.task('setupEmu');
+            cy.task('setupEmu', { mnemonic: 'all all all all all all all all all all all all' });
             cy.task('startBridge');
-
+            cy.task('metadataStartProvider', provider);
             if (!keepDB) {
                 cy.resetDb();
             }
-
             const { baseUrl } = Cypress.config();
-            const assetPrefix = Cypress.env('ASSET_PREFIX');
-            cy.visit(baseUrl + testUrl + assetPrefix);
+
+            //
+            // Test execution
+            //
+
+            // const assetPrefix = Cypress.env('ASSET_PREFIX');
+            // cy.visit(baseUrl + testUrl + assetPrefix);
+            cy.visit(baseUrl + testUrl, {
```

Comment:
Please add comment like this:
// This test is not intended to run against local dev server. Yoy should change base url to sldev 
// For docker, go to `docker-compose.suite-test.yml` and set `CYPRESS_baseUrl=https://suite.corp.sldev.cz/suite-web/` env variable